ddtrace/tracer/telemetry.go

```diff
@@ -0,0 +1,76 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package tracer
+
+import (
+	"fmt"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/telemetry"
+)
+
+// start the global instrumentation telemetry client with tracer data
+func startTelemetry(c *config) {
+	// need to re-intialize default values
+	if !telemetry.GlobalClient.Started() {
+		telemetry.GlobalClient.Default()
+		telemetry.GlobalClient.ApplyOps(
+			telemetry.WithService(c.serviceName),
+			telemetry.WithEnv(c.env),
+			telemetry.WithHTTPClient(c.httpClient),
+			// c.logToStdout is true if serverless is turned on
+			telemetry.WithURL(c.logToStdout, c.agentURL.String()),
+			telemetry.WithVersion(c.version),
+		)
+	}
+	telemetryConfigs := []telemetry.Configuration{
+		{Name: "trace_debug_enabled", Value: c.debug},
+		{Name: "agent_feature_drop_p0s", Value: c.agent.DropP0s},
+		{Name: "stats_computation_enabled", Value: c.agent.Stats},
+		{Name: "dogstatsd_port", Value: c.agent.StatsdPort},
+		{Name: "lambda_mode", Value: c.logToStdout},
+		{Name: "send_retries", Value: c.sendRetries},
+		{Name: "trace_startup_logs_enabled", Value: c.logStartup},
+		{Name: "service", Value: c.serviceName},
+		{Name: "universal_version", Value: c.universalVersion},
+		{Name: "env", Value: c.env},
+		{Name: "agent_url", Value: c.agentURL.String()},
+		{Name: "agent_hostname", Value: c.hostname},
+		{Name: "runtime_metrics_enabled", Value: c.runtimeMetrics},
+		{Name: "dogstatsd_addr", Value: c.dogstatsdAddr},
+		{Name: "trace_debug_enabled", Value: c.noDebugStack},
```

Comment:
Just double checking that this isn't backwards? The name is `trace_debug_enabled` but the value `noDebugStack` looks like it will be true if debugging _isn't_ enabled?

Reply:
Yep its backwards, thanks for catching this!